omdb/models.py

```diff
@@ -80,6 +80,7 @@ class Item(Storage):
         'Runtime',
         'Season',
         'seriesID',
+        'totalSeasons',
```

Comment:
Looks like `totalResults` was also added according to http://www.omdbapi.com/ changes for 1/20/16.

Care to add that too? I don't have an apikey so I'm not able to view/verify what the search results look like now :frowning_face:.